### ISSUE TITLE
User projects access

### ISSUE THREAD

**Issue by hkff** [Codiad/Codiad#597]
In Project Access panel, add an option to allow a user to access to his projects only (created by him).

**Comment by viharm** [comment]
Yes, I would like this too. A separate workspace for each user would be a good start, even if it is subdirectories inside the workspace directory
